assignments/prototypes.js

```diff
@@ -33,6 +53,19 @@
   * should inherit takeDamage() from CharacterStats
 */
 
+Humanoid.prototype = Object.create(CharacterStats.prototype);
+function Humanoid(features) {
+  CharacterStats.call(this, features);
+  this.team = features.team;
+  this.weapons = features.weapons;
+  this.language = features.language;
+}
+
+Humanoid.prototype.greet = function () {
+  return `${this.name} offers a greeting ${this.language}`
+}
+
+
```

Comment:
This looks great! I'd love for you to go look at the solution code to see how they organized the code a little bit differently, but this works just fine, and as long as it works, it's perfect :) great job, Erik.